src/EnergyPlus/DataVectorTypes.hh

```diff
@@ -85,15 +85,14 @@ namespace DataVectorTypes {
     //  subscript lookup and are templates so we are using them as plug replacements
     //  for consistent API and to avoid cost of copying them
     // Note: For vectorization contexts std::array is a better choice
-    typedef ObjexxFCL::Vector3<Real64> Vector;
-    typedef ObjexxFCL::Vector2<Real64> Vector_2d;
+    using Vector = ObjexxFCL::Vector3<Real64>;
```

Comment:
I do think this is really useful in that case. 
* I do prefer `DataVectorTypes::Vector` versus  `ObjexxFCL::Vector3<Real64>`
* If in function (and not at global scope) I want to further reduce my typing, I could do `using namespace DataVectorTypes` and be good to go, instead of having to that plus `using namespace ObjexxFCL`
* `Vector` is clearer and shorter than `ObjexxFCL::Vector3<Real64>`, and we have to put (and read) that all around. Templated types are harder to read and type.
* You cannot mess up the template call. Vector is a template specialization, there's no room for error. If you leave people having to do `Vector3<Real64>`, there is room for doing stuff like `Vector3<float>` by mistake or something.

`std::vector<Vector>`doesn't look that bad to me, but I see your point. Naming is perhaps the issue here. Calling it `Vector3d` would solve that and be much more explicit (fwiw, the OS SDK has a Vector3d and a Point3d class)

----

Note: I didn't introduce it.

 but `using T2 = T1` is the more modern way of doing `typedef T1 T2`: it's less ambiguous, clang tidy has a [modernize-use-using](https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-using.html) check for that.

Reply:
I see that you didn't really change anything here, just transformed a typedef to a using.  Depending on how many times you have to write it out, I'm not opposed to the full length `ObjexxFCL::Vector3<Real64>` declaration right where it's used.  If it's dozens of times or more, than maybe a using is nice.  

I didn't actually know that the `using` approach was the modern way to do a typedef, but it makes sense -- I have the same issue with typedef and never remembering the order.  

I think what you have done is technically neutral, so I don't think it's worth worrying about too much, but if you have to make more changes, maybe we can evaluate whether we should just undo this change and reassess later, or maybe propagate another pattern through.